src/v/kafka/server/tests/group_test.cc

```diff
@@ -51,6 +51,7 @@ static group get() {
       conf,
       nullptr,
       make_backward_compatible_serializer(),
+      model::topic_namespace{model::ns{"kafka"}, model::topic{"sample"}},
```

Comment:
This change was removed since I also removed the namespace label from committed_offset metric.

Changes were pushed to a separate PR at https://github.com/redpanda-data/redpanda/pull/5349